network_simulator/link.py

```diff
@@ -95,3 +164,118 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+
+    def get_link_cost(self):
+        """
+        Returns link cost. For now, just returns static propagation delay.
+        :return: float of link delay time in seconds
+        """
+
+        # TODO consider average wait time for last N packets
```

Comment:
(comment doesn't require response) just wanted to note that we will have to add queuing delay at some point (since if we didn't routing table updates wouldn't be that useful). I don't think it's extra credit.